test/kuttl/test-user-defined-ns/00-cert-manager.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands: 
+- command: kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.0/cert-manager.yaml
```

Comment:
This version differs from the one we deploy otherwise (1.5.3). Also, 1.6.0 introduced an upgrade regression which 1.6.1 fixes.

Reply:
Note: there's no check that verifies that cert-manager was started and correctly deployed. This causes errors in faster machines and works on slower machines. The same issue plagues cass-operator a bit, on my Linux workstation the execution of Kube commands is so fast that the integration-tests fail since the webhook wasn't enough quick to initialise, but the tests work fine in GHA and my Mac since they're slower. You could run into the same issue here (unless kettle is so slow that it can't).

Reply:
It doesn't hurt to test multiple cert-manager versions IMO. I've never seen this test flake but if you find it does I'm happy to hear about it. 

> Note: there's no check that verifies that cert-manager was started and correctly deployed

Appreciate the tip here (timing issues are hugely painful and it is great that we're all looking out for them.) In this case `00-assert.yaml` actually does ensure that all cert-manager components are successfully deployed and available prior to proceeding to the second step.